cmd/clusterawsadm/cloudformation/bootstrap/managed_control_plane.go

```diff
@@ -31,5 +31,5 @@ func (t Template) eksControlPlanePolicies() []string {
 }
 
 func eksAssumeRolePolicy() *iamv1.PolicyDocument {
```

Comment:
Maybe give this a more specific function name, or inline it completely.

Reply:
Good idea, this has been inlined.